workflow/workflow_test.go

```diff
@@ -968,3 +968,54 @@ func TestWorkflowWithEscapedCharacters(t *testing.T) {
 
 	}
 }
+
+var workflowWithOutputSchemaBad = `
+version: v0.2.0
+input:
+  root: RootObject
+  objects:
+    RootObject:
+      id: RootObject
+      properties: {}
+steps:
+  long_wait:
+    plugin:
+      src: "n/a"
+      deployment_type: "builtin"
+    step: wait
+    input:
+      wait_time_ms: 1
+outputs:
+  success:
+    first_step_output: !expr $.steps.long_wait.outputs
+outputSchema:
+  success:
+    schema:
+      root: RootObjectOut
+      objects: 
+        RootObjectOut: 
+          id: RootObjectOut
+          properties: {}`
+
+func TestWorkflow_Excute_Error_OutputSchema(t *testing.T) {
```

Comment:
ah I thought our `yaml` code was in the engine package b/c the [FromYAML() with its call to workflowSchema.Unserialize(rawWorkflow) seemed to be at the root of the cause of the error](https://github.com/arcalot/arcaflow-engine/blob/da06fdeb14ed275d46facd4db3e7f1d63e6679f3/workflow/yaml.go#L45) and FromYAML() is only ever called in the engine package.  

Since `FromYAML()` is in the workflow package, it makes sense to me to move the test to `workflow_test`.